PrebidMobile/PrebidMobileRendering/Prebid/PBMCore/PBMPrebidParameterBuilder.m

```diff
@@ -67,8 +67,13 @@ - (void)buildBidRequest:(nonnull PBMORTBBidRequest *)bidRequest {
     BOOL const isHTML = ([adFormats containsObject:AdFormat.banner]);
     BOOL const isInterstitial = self.adConfiguration.adConfiguration.isInterstitialAd;
     
+    NSString *requestID = self.sdkConfiguration.auctionSettingsId;
```

Comment:
Would be great to have a test when both `auctionSettingsId` and `prebidServerAccountId` are present.